panoramix/assets/webpack.config.js

```diff
@@ -30,9 +30,14 @@ var config = {
         include: APP_DIR,
         loader: "style-loader!css-loader"
       },
-    /* for require('*.less') */
+    /* for css linking images */
+      { test: /\.png$/, loader: "url-loader?limit=100000" },
+      { test: /\.jpg$/, loader: "file-loader" },
+      { test: /\.gif$/, loader: "file-loader" },
+    /* for font-awesome */
       { test: /\.woff(2)?(\?v=[0-9]\.[0-9]\.[0-9])?$/, loader: "url-loader?limit=10000&minetype=application/font-woff" },
```

Comment:
oh I thought it was an attempt at getting `font-awesome` to work as the failure is around `woff` files and such...